integration/src/test/kotlin/com/spectralogic/tpfr/integration/IntegrationTests.kt

```diff
@@ -16,135 +16,154 @@
 package com.spectralogic.tpfr.integration
 
 import com.spectralogic.tpfr.api.ServerServiceFactoryImpl
-import com.spectralogic.tpfr.client.Client
-import com.spectralogic.tpfr.client.ClientImpl
+import com.spectralogic.tpfr.client.TpfrClient
+import com.spectralogic.tpfr.client.TpfrClientImpl
 import com.spectralogic.tpfr.client.model.*
 import org.assertj.core.api.KotlinAssertions.assertThat
 import org.junit.BeforeClass
 import org.junit.Test
 import java.time.LocalTime
+import java.util.*
 import java.util.concurrent.ForkJoinPool
 import java.util.concurrent.TimeUnit
+
 class IntegrationTests {
 
     companion object {
 
-        private val origFilesPath = "\\\\ISV_RETROSPECT1\\Users\\orig\\"
-        private val restoredFilesPath = "\\\\ISV_RETROSPECT1\\Users\\restored\\"
-        private lateinit var client: Client
-        private val numberOfThreads = 12
+        private const val origFilesPath = "\\\\eng-dell-26\\temp\\media\\BP-PFR\\toIndex\\"
+        private const val restoredFilesPath = "\\\\eng-dell-26\\temp\\media\\BP-PFR\\restoredPartial\\"
+        private const val restoredFragmentPath = "\\\\eng-dell-26\\temp\\media\\BP-PFR\\restoredFragment\\"
+        private lateinit var tpfrClient: TpfrClient
+        private const val numberOfThreads = 12
 
-        @BeforeClass @JvmStatic
+        @BeforeClass
+        @JvmStatic
         fun startup() {
-            val endpoint = "http://10.85.41.78:60792"
+            val endpoint = "http://eng-dell-26.eng.sldomain.com:60792"
```

Comment:
Should this be pulled from an environment variable?

Reply:
We can, i'll create a diff Jira for that